<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>932</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="startbuttom">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP0_0">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP0_1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP0_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP0_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP1_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP1_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>90</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP1_0">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP1_1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP2_1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP2_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>130</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP2_0">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP2_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>130</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP3_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>170</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP3_1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP3_0">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>170</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP3_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>170</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP4_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>210</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP4_0">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>210</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP4_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>210</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP4_1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>210</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP6_0">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>290</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP6_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>290</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP8_1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>370</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP5_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>250</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP8_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>370</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP6_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>290</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP6_1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>290</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP9_1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>410</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP7_0">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>330</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP8_0">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>370</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP5_0">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>250</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP9_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>410</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP5_1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>250</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP7_1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>330</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP9_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>410</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP7_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>330</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP8_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>370</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP7_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>330</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP9_0">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>410</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP5_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>250</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP0_4">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>50</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP4_4">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>210</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP1_4">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>90</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP2_4">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>130</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP3_4">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>170</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP9_4">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>410</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP6_4">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>290</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP5_4">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>250</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP8_4">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>370</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP7_4">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>330</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="IP0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">IP0</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="IP1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>IP1</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="IP2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>IP2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="IP3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>IP3</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="IP4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>IP4</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="IP8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>IP8</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="IP9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>IP9</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="IP7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>IP7</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="IP6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>IP6</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="IP5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>IP5</string>
    </property>
   </widget>
   <widget class="QLabel" name="connect0">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>50</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>not connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="connect1">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>90</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>not connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="connect2">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>130</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>not connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="connect3">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>170</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>not connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="connect4">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>210</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>not connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="connect7">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>330</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>not connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="connect8">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>370</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>not connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="connect9">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>410</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>not connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="connect5">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>250</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>not connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="connect6">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>290</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>not connect</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP0_5">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>50</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP0_6">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>50</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP0_7">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>50</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP1_5">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>90</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP1_6">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>90</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP1_7">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>90</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP2_5">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>130</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP2_6">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>130</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP2_7">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>130</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP3_5">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>170</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP3_6">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>170</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP3_7">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>170</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP4_5">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>210</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP4_6">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>210</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP4_7">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>210</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP5_5">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>250</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP5_6">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>250</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP5_7">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>250</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP6_5">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>290</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP6_6">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>290</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP6_7">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>290</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP7_5">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>330</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP7_6">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>330</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP7_7">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>330</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP8_5">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>370</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP8_6">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>370</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP8_7">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>370</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP9_5">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>410</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP9_6">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>410</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="IP9_7">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>410</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>932</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
